ouroboros-consensus-shelley/ouroboros-consensus-shelley.cabal

```diff
@@ -69,6 +69,7 @@ library
                      , deepseq
                      , measures
                      , mtl               >=2.2   && <2.3
+                     , microlens
```

Comment:
microslens < mtl by alphabet :)

Reply:
I can't sort by more than the first letter in my mind, I need a program for that :stuck_out_tongue_winking_eye: 

Fixed